x-pack/plugins/security_solution/public/flyout/document_details/right/components/assignees.tsx

```diff
@@ -91,25 +98,64 @@ export const Assignees: FC<AssigneesProps> = memo(
       setIsPopoverOpen((value) => !value);
     }, []);
 
-    const onAssigneesApply = useCallback(
-      async (assigneesIds: AssigneesIdsSelection[]) => {
+    const handleApplyAssignees = useCallback(
```

Comment:
I don't know if I missed that in a past review and if I did not review that code, but this seems pretty complex. We have on callback that triggers two other callbacks which then calls a function passed via props so that the parent can do some action in another callback...

I understand that it's out of the scope of this PR, but I think this should be revisited in the future to make things simpler. I don't think the parent should be aware or anything its child is doing, the refetch can be done here.